src/System.Drawing.Common/src/System/Drawing/Font.Unix.cs

```diff
@@ -224,27 +84,20 @@ public static Font FromHfont(IntPtr hfont)
                 newStyle |= FontStyle.Bold;
             }
 
-            if (lf.lfHeight < 0)
-            {
-                newSize = lf.lfHeight * -1;
-            }
-            else
-            {
-                newSize = lf.lfHeight;
-            }
-
-            return (new Font(newObject, lf.lfFaceName, newStyle, newSize));
+            return new Font(newObject, lf.lfFaceName, newStyle, Math.Abs(lf.lfHeight));
         }
 
         public IntPtr ToHfont()
         {
-            if (fontObject == IntPtr.Zero)
-                throw new ArgumentException("Object has been disposed.");
+            if (_nativeFont == IntPtr.Zero)
+            {
+                throw new ArgumentException(SR.ObjectDisposed);
```

Comment:
This is the same exception type and message as on Windows?

Reply:
I just tried it and I got:
https://github.com/dotnet/corefx/blob/026fdf972519c786c77f20689c5bea4c04524ecf/src/System.Drawing.Common/src/Resources/Strings.resx#L168-L170
as an error message.  Assuming I didn't do something wrong, we shouldn't be creating a new resource for this.

Reply:
Fixed